applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/status.js

```diff
@@ -0,0 +1,270 @@
+				E('th', { 'class': 'th' }, _('bssid')),
+				E('th', { 'class': 'th' }, _('ssid')),
+				E('th', { 'class': 'th' }, _('freq')),
+				E('th', { 'class': 'th' }, _('n assoc')),
+				E('th', { 'class': 'th' }, _('noise')),
+				E('th', { 'class': 'th' }, _('load')),
+				E('th', { 'class': 'th' }, _('max assoc')),
+				E('th', { 'class': 'th' }, _('roam src')),
+				E('th', { 'class': 'th' }, _('roam tgt')),
+				E('th', { 'class': 'th' }, _('rrm_nr mac')),
+				E('th', { 'class': 'th' }, _('rrm_nr hex'))
+				
+			])
+		]);
+	
+		var connectioninfo_table_entries =[];
+		for(var wlan in Localinfo) {
+			connectioninfo_table_entries.push([
+							'<nobr>'+wlan+'</nobr>', 
+							Localinfo[wlan]['bssid'],
+							Localinfo[wlan]['ssid'],
+							Localinfo[wlan]['freq'],
+							Localinfo[wlan]['n_assoc'],														
+							Localinfo[wlan]['noise'],														
+							Localinfo[wlan]['load'],														
+							Localinfo[wlan]['max_assoc'],														
+							Localinfo[wlan]['roam_events']['source'],														
+							Localinfo[wlan]['roam_events']['target'],														
+							Localinfo[wlan]['rrm_nr'][0],																					
+							Localinfo[wlan]['rrm_nr'][2]		
+						]);
+				}	
+		for(var wlan in Remoteinfo) {
+			connectioninfo_table_entries.push([
+							'<nobr>'+wlan+'</nobr>', 
+							Remoteinfo[wlan]['bssid'],
+							Remoteinfo[wlan]['ssid'],
+							Remoteinfo[wlan]['freq'],
+							Remoteinfo[wlan]['n_assoc'],														
+							Remoteinfo[wlan]['noise'],														
+							Remoteinfo[wlan]['load'],														
+							Remoteinfo[wlan]['max_assoc'],														
+							Remoteinfo[wlan]['roam_events']['source'],														
+							Remoteinfo[wlan]['roam_events']['target'],														
+							Remoteinfo[wlan]['rrm_nr'][0],																					
+							Remoteinfo[wlan]['rrm_nr'][2]		
+
+						]);
+				}	
+		cbi_update_table(connectioninfo_table, connectioninfo_table_entries, E('em', _('No data')));
+		body.appendChild(connectioninfo_table);
+/////////////////////////////////////////
+
+
+		var compactconnectioninfo_table = E('table', { 'class': 'table cbi-section-table' }, [
+				E('tr', { 'class': 'tr table-titles' }, [
+					E('th', { 'class': 'th' }, _('wlan')),
+					E('th', { 'class': 'th' }, _('ssid')),
+					E('th', { 'class': 'th' }, _('freq')),
+					E('th', { 'class': 'th' }, _('load')),
+					E('th', { 'class': 'th' }, _('n')),
+					E('th', { 'class': 'th' }, _('host'))
+					
+				])
+			]);
+		var compactconnectioninfo_table_entries =[];
+		for(var wlan in Localinfo) {
+			var hostl=''
+			for(var mac in Clients) {
+				for(var wlanc in Clients[mac]) {
+					if ((String(Clients[mac][wlanc]['connected']).valueOf()==String("true").valueOf()) && 
+					   (wlan===wlanc)) {
+							var foundname=mac;
+							for(var mac2 in hosts) {								
+								if ((String(mac).toLowerCase()).valueOf()==(String(mac2).toLowerCase()).valueOf()) {
+									foundname=hosts[mac2]['ipaddrs'][0];
```

Comment:
I don't know why, but for one of my clients, the `ipaddrs` is empty. It has one IPv4, but for some reason it is not "detected" by luci. This produces an `undefined` in the list of hosts.
![image](https://github.com/openwrt/luci/assets/2673520/1ec789cf-b780-4218-8783-da5c03598b13)

The `ip6addrs` exists, I don't know why the `ipaddrs` or the `name` are empty. I installed this application in an AP to test it, so maybe it has not retrieved all the info, but is clear that is a case that happens sometimes.

Reply:
Two ways i could fix this, either check if the IPv4 exists, if not, then display the mac. Or display the IPv6 instead (if that exists). Any preference?

Reply:
The problem with IPv6 is that there are different addresses, and will be difficult to find one that can be useful.  So I think is better to simply publish the mac in this case.

Reply:
ok done

Reply:
ok done